docs/examples/TabsNoAnimation.js

```diff
@@ -0,0 +1,10 @@
+const tabsInstance = (
+  <Tabs defaultActiveKey={1} animation={false}>
+    <Tab eventKey={1} title='Tab 1'>Tab 1 content</Tab>
+    <Tab eventKey={2} title='Tab 2'>Tab 2 content</Tab>
+    <Tab eventKey={3} title='Tab 3' disabled>Tab 3 content</Tab>
+
```

Comment:
unfortunate extraneous white line! But we can fix it right after. I don't think it's preventing from merge